### PR TITLE
updated docs

### PR DESCRIPTION
**Pull request by kapv89**

### CODE REVIEW

File: go/database/sql/README.md [hunk 1]
```diff
@@ -40,15 +39,24 @@ type CommenterOptions struct {
 	EnableController  bool   // applicable for web frameworks
 	EnableAction      bool   // applicable for web frameworks
 	EnableApplication bool   // applicable for web frameworks
-	Application       string // user-provided application-name. optional
-	}
+}
+```
+
+Users can also provide static tags they want to use by using `core.StaticTags` struct. These tags are optional
```
Comment (sjs994) [inline go/database/sql/README.md:45]: Maybe we should sometime describe how these static tags helps. There is some logic in the calculation part regarding both Application and DBDriver, where the library will try to find these tags if not present.

File: go/gorrila/mux/README.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+	"github.com/gorilla/mux"
+)
+
+func runApp() {
+	r := mux.NewRouter()
+	r.Use(sqlcommentermux.SQLCommenterMiddleware)
+
+	r.HandleFunc("/", ActionHome).Methods("GET")
+
+    http.ListenAndServe(":8081", r)
```
Comment (sjs994) [inline go/gorrila/mux/README.md:32]: indentation fix.

File: go/gorrila/mux/README.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+
+## Example
+
+```go
+import (
+    "net/http"
+
+    sqlcommentermux "github.com/google/sqlcommenter/go/gorrila/mux"
+	"github.com/gorilla/mux"
```
Comment (sjs994) [inline go/gorrila/mux/README.md:23]: indentation fix

File: go/gorrila/mux/README.md (hunk 2 above)
Comment (sjs994) [reply go/gorrila/mux/README.md:32]: Please look at other places where indentation needs to be fixed.

**Review by sjs994 — APPROVED** [review]

File: go/gorrila/mux/README.md (hunk 3 above)
Comment (kapv89) [reply go/gorrila/mux/README.md]: fixed

File: go/gorrila/mux/README.md (hunk 2 above)
Comment (kapv89) [reply go/gorrila/mux/README.md:32]: fixed

File: go/database/sql/README.md (hunk 1 above)
Comment (kapv89) [reply go/database/sql/README.md]: fixed

**Pull request merged**